src/my_plugin_ios/pubspec.yaml

```diff
@@ -25,4 +25,4 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   plugin_platform_interface: ^2.0.0
-  very_good_analysis: ^3.0.0
+  very_good_analysis: ^3.0.1
```

Comment:
missing the anysis option file for both android and ios?

Reply:
resolved ✅